assets/pages/regular-contributions/helpers/ajax.js

```diff
@@ -201,6 +203,7 @@ function statusPoll(
 
   if (pollCount >= MAX_POLLS) {
     successfulConversion(participations);
+    ophan.sendInitialEvent(getAbsoluteURL(routes.recurringContribThankyou));
```

Comment:
Great, since this is a new feature I think you need to update the version of ophan in support's `package.json`